### PR TITLE
feat: add support to change default owner and group when calling jenkins_plugin module

### PR DESCRIPTION
**Pull request by Jordi-Pineiro**
The role can be used with Jenkins running on a docker container

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,6 +120,14 @@ The update center proxy no proxy hosts list.
 Controls if the initial setup of the jenkins instance should be completed.
 This may be necessary in proxy environments where the jenkins instance still thinks it is in offline mode.
 
+    jenkins_plugins_owner: "jenkins"
+
+User owning the files in the plugins folder
```
Comment (tobias-richter) [inline README.md:125]: Just a small thing, `.` is missing at the end.

File: README.md [hunk 2]
```diff
@@ -120,6 +120,14 @@ The update center proxy no proxy hosts list.
...
+
+    jenkins_plugins_group: "jenkins"
+
+Group owning the files in the plugins folder
```
Comment (tobias-richter) [inline README.md:129]: Just a small thing, `.` is missing at the end.

**Review by tobias-richter — CHANGES_REQUESTED** [review]
Looks nearly perfect. Please add the missing `.` and then we are ready to go.

File: README.md (hunk 1 above)
Comment (Jordi-Pineiro) [reply README.md]: Fixed. Thanks!!!

**Review by tobias-richter — APPROVED** [review]

**Pull request merged**